### PR TITLE
application: serial_lte_modem: Update #XSLMVER command

### PR DESCRIPTION
**Pull request by junqingzou**
Add to report the version of the modem library.
Correct mistakes in doc for this command.

Signed-off-by: Jun Qing Zou <jun.qing.zou@nordicsemi.no>

### CODE REVIEW

**Review by VTPeltoketo — APPROVED** [review]

File: applications/serial_lte_modem/doc/Generic_AT_commands.rst [hunk 1]
```diff
@@ -12,12 +12,12 @@ The following commands list contains generic AT commands.
 SLM version #XSLMVER
 ====================
 
-The ``#XSLMVER`` command requests the SLM version.
+The ``#XSLMVER`` command requests the versions of SDK in which the SLM application is built.
```
Comment (wiba-nordic) [inline applications/serial_lte_modem/doc/Generic_AT_commands.rst:15]:
```suggestion
The ``#XSLMVER`` command requests the version of the |NCS| in which the SLM application is built.
```
Should this mention the modem library as well?

File: applications/serial_lte_modem/doc/Generic_AT_commands.rst [hunk 2]
```diff
@@ -12,12 +12,12 @@ The following commands list contains generic AT commands.
...
 
 Set command
 -----------
 
-The set command requests the SLM version.
+The set command requests the version of the NCS SDK and that of the modem library.
```
Comment (wiba-nordic) [inline applications/serial_lte_modem/doc/Generic_AT_commands.rst:20]:
```suggestion
The set command requests the version of the |NCS| and that of the modem library.
```
The S in NCS is SDK, so NCS SDK is redundant.

File: applications/serial_lte_modem/doc/Generic_AT_commands.rst [hunk 3]
```diff
@@ -31,19 +31,21 @@ Response syntax
 
 ::
 
-   #XSLMVERSION: <version>
+   #XSLMVER: <ncs_version>,<libmodem_version>
 
-The ``<version>`` value returns a string containing the SLM version.
+The ``<ncs_version>`` value returns a string containing the version of the NCS SDK.
```
Comment (wiba-nordic) [inline applications/serial_lte_modem/doc/Generic_AT_commands.rst:36]:
```suggestion
The ``<ncs_version>`` value returns a string containing the version of the |NCS|.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by SeppoTakalo — APPROVED** [review]

File: applications/serial_lte_modem/doc/Generic_AT_commands.rst (hunk 1 above)
Comment (junqingzou) [reply applications/serial_lte_modem/doc/Generic_AT_commands.rst:15]: Added a line

**Review by wiba-nordic — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Review by MarkusLassila — APPROVED** [review]

**Comment by tmon-nordic** [comment]
It seems that this should have changelog entry because it changes existing command output. Also, I think bug label should be added to this PR.

**Comment by wiba-nordic** [comment]
Good point about changing outputs. Agreed.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -256,6 +256,7 @@ nRF9160: Serial LTE modem
 * Updated:
 
   * The response for the ``#XDFUGET`` command, using unsolicited notification to report download progress.
+  * Report the version of |NCS| as well as the modem library in ``#XSLMVER``.
```
Comment (wiba-nordic) [inline doc/nrf/releases/release-notes-changelog.rst:259]:
```suggestion
  * The ``#XSLMVER`` command to report the versions of both the |NCS| and the modem library.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Comment by wiba-nordic** [comment]
The label isn't required when an changelog entry has been added.

**Pull request merged**